### DISCUSSION TITLE
Feature request: Clickable tags

### DISCUSSION THREAD

**Discussion by MaxWasUnavailable** [subrotokumar/kurumi#5] [Ideas]
Not sure if this is within scope, but it'd be nice to be able to click on tags to be directed to a search / sorted list view of content with those same tags.

An example would be clicking on the isekai tag on an anime, and being met with a list of isekai.

**Comment by subrotokumar** [comment]
Hello @MaxWasUnavailable,

Thank you for sharing your feature request! We appreciate your input and your interest in enhancing the functionality of our app.

Your suggestion to make tags clickable for a sorted list view of content with the same tags is indeed intriguing and aligns well with the goal of improving user experience. This could significantly enhance navigation and discovery within the app.

While I can't make any immediate promises, I want you to know that we take every user suggestion seriously. Your feedback is invaluable as it helps us prioritize features that could benefit our users the most.

Please continue to share any other ideas or suggestions you might have. Our aim is to create an app that's both functional and user-friendly, and your contribution plays a significant role in achieving that goal.

**Comment by subrotokumar** [comment]
I'm excited to share that your suggested feature of Clickable Tag and Genre Buttons has been planned for our upcoming version v1.4.0.
Update Release Date: 15-08-2023 6:00 AM IST
